extensions/grpc/runtime/src/main/java/io/quarkus/grpc/runtime/config/GrpcServerConfiguration.java

```diff
@@ -73,4 +73,10 @@
      */
     @ConfigItem(defaultValue = "1")
     public int instances;
+
+    /**
+     * Configures the netty server settings.
+     */
+    @ConfigItem
+    public GrpcServerNettyConfig nettyConfig;
```

Comment:
I have no opinion about this patch, I will let it to the experts, but this should be named `netty`, not `nettyConfig`. We don't want `-config` in the property name.

Reply:
`netty` might be ok.